node/node.go

```diff
@@ -95,12 +96,20 @@ func (n *Node) Start(ctx context.Context) error {
 			return err
 		}
 	}
+	// start block service if not Light node
+	if n.Type != Light {
+		err = n.BlockServ.Start(ctx)
+		if err != nil {
+			log.Errorw("starting block service", "err", err)
+			return fmt.Errorf("node: failed to start block service: %w", err)
+		}
+	}
```

Comment:
Hmmm. The combinatorics here look like it would get messy (case in point, adding a Dev mode means you have to change "Full" to "not Light," which is difficult to reason about). Instead, could you have each node type set up a list of services themselves?

Not necessary to change in this PR.

Reply:
Yeah eventually we'll make fx DI lib handle all this -- #134